src/cpp/session/modules/SessionGit.cpp

```diff
@@ -234,8 +234,7 @@ std::string gitText(const ShellArgs& args)
    else
       ss << s_gitExePath << " ";
 
-   std::string arguments = core::algorithm::join(args, " ");
-   ss << arguments << "\n";
+   ss << core::algorithm::join(shellArgs.args(), " ") << "\n";
```

Comment:
Is this necessary b/c of the change to the implementation of `join()` above? If so, are there other places we might need to change? (Makes me just a little nervous to mess with the implementation of a core algorithm so close to release...)

Reply:
That's correct. Previously we were relying on an implicit conversion from ShellArgs to std::vector<std::string>; with the more permissive template `join()` implementation that conversion never gets a chance to happen, so we need to be explicit here.

I definitely understand the hesitance to touch this code though so I'll make a more narrowly-scoped change.